lib/ansible/plugins/inventory/azure_rm.py

```diff
@@ -380,7 +382,7 @@ def _process_queue_batch(self):
                     query_parameters = {'api-version': item.api_version}
                     req = self._client.get(item.url, query_parameters)
 
-                    batch_requests.append(dict(httpMethod="GET", url=req.url))
+                    batch_requests.append(dict(httpMethod="GET", url=req.url, name="ansible" + str(random.randint(1, 1000000))))
```

Comment:
It seems strange to see `random.randint(1, 1000000)`. Obviously, I'd expect that to lead to a 1-in-million collision of names. Why wouldn't you just use alternatives:

 - a simple iterator within the loop where requests are made
 - just call uuid4() to get a globally unique string

Reply:
resolved